### ISSUE TITLE
(DRAFT) Support interoperability with Interledger

### ISSUE THREAD

**Issue by sbellem** [bigchaindb/bigchaindb#1963]
-Details will be added soon._

Back in June (2017) that meant supporting hash locks (sha256 conditions).

**Comment by ttmc** [comment]
Issue #725 may be related.

**Comment by diminator** [comment]
Indeed, this means that one should always sign the transaction - rather than assuming the input-fulfillment has a signature forced

**Comment by emschwartz** [comment]
Out of curiosity, how long do BigchainDB transactions take to process? Also, do you guys have any plans to support some kind of payment channels (or would that be outside the scope of your use cases)?

**Comment by ttmc** [comment]
> Out of curiosity, how long do BigchainDB transactions take to process?

Once we've integrated Tendermint (i.e. in the next month or so), BigchainDB transaction finality should take around a second or so, for a globally-distributed network.

> Also, do you guys have any plans to support some kind of payment channels (or would that be outside the scope of your use cases)?

I don't know of any plans to support any kind of payment channels.
